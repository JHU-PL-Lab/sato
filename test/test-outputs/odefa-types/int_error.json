[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "x" ],
          "err_match_val": "false",
          "err_match_expected": "int",
          "err_match_actual": "bool"
        }
      ],
      [
        "Error_match",
        {
          "err_match_aliases": [ "y" ],
          "err_match_val": "false",
          "err_match_expected": "int",
          "err_match_actual": "bool"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "z = x + y",
    "err_steps": 68
  }
]