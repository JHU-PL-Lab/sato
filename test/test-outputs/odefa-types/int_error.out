Type errors for:
- Input sequence  : []
- Found at clause : z = x + y
--------------------
* Value    : x = false
* Expected : int
* Actual   : bool
--------------------
* Value    : y = false
* Expected : int
* Actual   : bool
2 errors found. No further control flows exist.
